lib/compiler.js

```diff
@@ -1,9 +1,13 @@
 var hasOwn = Object.prototype.hasOwnProperty;
 var assert = require("assert");
-var types = require("ast-types");
+var types = require("ast-types/fork")([
+  require("ast-types/def/esprima"),
+  require("ast-types/def/babel6")
+]);
```

Comment:
esprima loads es7, and es7 loads es6, and es6 loads core.
babel6 loads babel and flow.